data/data/bootstrap/files/usr/local/bin/approve-csr.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
```

Comment:
I generally start all scripts in [strict mode](http://redsymbol.net/articles/unofficial-bash-strict-mode/).

Reply:
This script needs to keep trying without failing _because there are going to be transient failures_

Reply:
You already have `Restart=on-failure`; this is about not trying to stumble forwards if something goes wrong in the middle.

(In this script it's probably fine...but I'd still have it on general principle)

Reply:
It's usually prefered to keep out services internal retrying rather than restart on failure..
https://github.com/openshift/installer/pull/1101 <- also moved away from strict _not exactly but same motivation_

Reply:
FWIW feel free to merge this PR as is.  I understand the motivation in #1101 but I have personally been badly burned in the past by having shell script not using strict mode.  Again it doesn't really apply to *this* script as is today so it's OK.